src/Routers/ClassesRouter.js

```diff
@@ -129,6 +129,7 @@ export class ClassesRouter extends PromiseRouter {
     this.route('GET', '/classes/:className', (req) => { return this.handleFind(req); });
     this.route('GET', '/classes/:className/:objectId', (req) => { return this.handleGet(req); });
     this.route('POST', '/classes/:className', (req) => { return this.handleCreate(req); });
+    this.route('PUT', '/classes/:className', (req) => { return this.handleUpdate(req); });
```

Comment:
yes it does :) https://github.com/ParsePlatform/parse-server/blob/master/src/PromiseRouter.js#L56